docker-compose.yml

```diff
@@ -13,25 +13,29 @@ services:
       - hasura
     build:
       context: ./registration
-      dockerfile: Dockerfile
+      dockerfile: Dockerfile.dev
```

Comment:
You shouldn't change the production environment values and it also does not relate the the PR.